pkg/grub/rootfs.cfg

```diff
@@ -188,7 +188,7 @@ function set_arm64 {
    set_global hv_console "console=dtuart sync_console"
    set_global hv_platform_tweaks " "
    set_global load_devicetree_cmd devicetree
-   set_global dom0_console "console=tty0 console=ttyS0,115200 console=hvc0"
+   set_global dom0_console "console=tty0 console=ttyAMA0,115200 console=hvc0"
```

Comment:
This needs to be conditioned on ARM vs. x86

Or maybe we can just keep both ttyAMA0 and ttyS0 -- after all -- multiple consoles are not that big of a deal for the kernel

Reply:
ttyS0 is there for 8250-compatible UARTs, do we have any on supported HW?

Reply:
Right -- didn't see that this is already in the set_arm64 function -- sorry for the noise @temerkhanov